### PR TITLE
Issue resolved "Handle the 0 Input in the Pressure sore and Pain Scale component"

### LINKED ISSUE #7365: Handle the 0 Input in the Pressure sore and Pain Scale component
**Describe the bug**

- [ ] In pressure sore, the weight and height input columns have inconsistent lengths.
image
- [ ] When we submit input with 0 on pressure sore / pain scale, both are accept empty input

**To Reproduce**

1. Go to the patient consultation page and click on log update.
2. Create a critical care log update then input 0 in the pain scale (Vital Section) 
3. save it and see the error.

similar error to pressure sore as well

**Expected behavior**

- [ ] In pressure sore, the weight and height column should have consistent length
- [ ] When we input 0 in the pain scale and submit it, the input should be skipped.
- [ ] similar behavior should be implemented in the pressure sore  as well

**Screenshots**

image
image

### PR DESCRIPTION
**Pull request by r-nikhilkumar**
- Fixes #7365

for width and height consistency:-

for PC:
image

for Mobile:
image

for the pressure 0 and painScale 0 issue:
you can check from git changed files

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res [hunk 1]
```diff
@@ -138,8 +138,12 @@ let make = (
...
                   type_="button"
                   onClick={e => {
-                    updatePart(state)
-                    hideModal(e)
+                    if(painScale==0.0){
+                      hideModal(e)
+                    }else{
+                      updatePart(state)
+                      hideModal(e)
+                    }
```
Comment (rithviknishad) [inline src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res]:
```suggestion
                    if (painScale == 0.0) {
                      hideModal(e)
                    } else {
                      updatePart(state)
                      hideModal(e)
                    }
```

File: src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreInputModal.res [hunk 2]
```diff
@@ -38,13 +38,17 @@ let make = (
   }, [state])
 
   let handleSubmit = e => {
-    updatePart(state)
-    hideModal(e)
     let region = PressureSore.regionToString(state.region)
-    if (state.length > 0.0 && state.width == 0.0) || (state.length == 0.0 && state.width > 0.0) {
+    if(state.length === 0.0 && state.width === 0.0){
```
Comment (rithviknishad) [inline src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreInputModal.res:42]:
```suggestion
    if(state.length === 0.0 && state.width === 0.0) {
```

**Review by rithviknishad — APPROVED** [review]
Use prettier to format code

**Review by khavinshankar — APPROVED** [review]
LGTM, check the linting once

**Review by Ashesh3 — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**